src/pages/Portfolio/LendingInsights.vue

```diff
@@ -0,0 +1,222 @@
+			<div class="tw-col-span-12 md:tw-col-span-6">
+				<kv-loading-placeholder v-if="loading" class="stat-placeholder" style="width: 4rem;" />
+				<dt v-show="!loading" class="stat-value">
+					{{ countryCount }}
+				</dt>
+				<dd class="stat-def">
+					Countries supported
+				</dd>
+				<router-link
+					to="/portfolio/lending-stats"
+					v-kv-track-event="['portfolio', 'click', 'countries-supported-details']"
+				>
+					Details
+				</router-link>
+			</div>
+			<div class="tw-col-span-12 md:tw-col-span-6">
+				<kv-loading-placeholder v-if="loading" class="stat-placeholder" style="width: 4rem;" />
+				<dt v-show="!loading" class="stat-value">
+					{{ inviteeLoanCount }}
+				</dt>
+				<dd class="stat-def">
+					Loans by your invitees
+				</dd>
+				<router-link
+					to="/portfolio/invites#myInviteActivity"
+					v-kv-track-event="['portfolio', 'click', 'invitee-loans-details']"
+				>
+					Details
+				</router-link>
+			</div>
+			<div class="tw-col-span-12 md:tw-col-span-6">
+				<kv-loading-placeholder v-if="loading" class="stat-placeholder" style="width: 10rem;" />
+				<dt v-show="!loading" class="stat-value">
+					{{ averageDonation }}
+				</dt>
+				<dd class="stat-def">
+					Average donation per $25 lent
+				</dd>
+				<button
+					class="tw-text-link"
+					@click="donationLightboxVisible = true"
+					v-kv-track-event="['portfolio', 'click', 'avg-donation-details']"
+				>
+					Details
+				</button>
+				<kv-lightbox
+					:visible="donationLightboxVisible"
+					title="Average donation per $25 lent"
+					@lightbox-closed="donationLightboxVisible = false"
+				>
+					<p class="tw-mb-1">
+						100% of every dollar lent on Kiva is used to fund loans. Kiva doesn't take a cut of the loans
+						made on our website or charge interest to our Lending Partners. Instead, we rely on donations
+						from our lender community to cover our operations and expand access to microfinance around the
+						globe. This is why we ask for an optional  donation each time you make a loan.
+					</p>
+					<p class="tw-mb-1">
+						Whether you are depositing funds to make a loan or relending money that a borrower has paid
+						back, please remember that Kiva can't exist without your support.
+					</p>
+					<template #controls>
+						<kv-button
+							to="/donate/supportus"
+							v-kv-track-event="['portfolio', 'click', 'avg-donation-lightbox-donate']"
+						>
+							Donate
+						</kv-button>
+					</template>
+				</kv-lightbox>
+			</div>
+			<div class="tw-col-span-12 md:tw-col-span-6">
+				<kv-loading-placeholder v-if="loading" class="stat-placeholder" style="width: 10rem;" />
+				<dt v-show="!loading" class="stat-value">
+					{{ amountLent }}
+				</dt>
+				<dd class="stat-def">
+					Total amount lent
+				</dd>
+				<router-link
+					to="/portfolio/loans"
+					v-kv-track-event="['portfolio', 'click', 'total-amount-lent-details']"
+				>
+					Details
+				</router-link>
+				<!-- <button
```

Comment:
Leave in commented out code?

Reply:
Yes, this was going to be part of this ticket but was separated into a follow-up ticket MARS-374.